pywr-schema/src/timeseries/mod.rs

```diff
@@ -461,6 +461,13 @@ impl TryFromV1<DataFrameParameterV1> for ConvertedTimeseriesReference {
                 Some(v) => v.as_str().map(|s| s.to_string()),
                 None => None,
             };
+            // remove the parse_dates for CSV files as this is already passed to read_csv in
```

Comment:
> ... already passed to read_csv ...

Would it be less surprising if this was not done automatically?

Reply:
@Batch21 thoughts on this before I merge these changes.

Reply:
Maybe less suprising but you're always going to need to parse dates for timeseries and doing it automatically will make model files a little more concise.